### PR TITLE
fix(renderer): attach `CommentNode`s to visual tree

### LINKED ISSUE #872: *ngIf not working in Angular application for hide/show layout after upgrade to NS3
_From @memphisvl on June 29, 2017 19:56_

It used to work on {N} 2.5 and now it seems that the only way to show/hide some elements is using visibility option.

**Before**
```html
<StackLayout class="page">
  <Label *ngIf="isPresent" text="{{showText}}"></Label>        
</StackLayout>
```

**Now (Workaround)**
```html
<StackLayout class="page">
  <Label visibility="{{isPresent ? 'visible' : 'collapse'}}" text="{{showText}}"></Label>        
</StackLayout>
```
Is this expected and from now on I will have to use visibility for UI logic?

**Versions**
```
    "@angular/animations": "~4.1.0",
    "@angular/common": "~4.1.0",
    "@angular/compiler": "~4.1.0",
    "@angular/core": "~4.1.0",
    "@angular/forms": "~4.1.0",
    "@angular/http": "~4.1.0",
    "@angular/platform-browser": "~4.1.0",
    "@angular/platform-browser-dynamic": "~4.1.0",
    "@angular/router": "~4.1.0",
    "moment": "^2.18.1",
    "nativescript-angular": "^3.1.0",
    "nativescript-angular-snapshot": "1.5.2-5.5.372.32",
    "nativescript-carousel-view": "^2.9.0",
    "nativescript-google-maps-sdk": "^2.3.0",
    "nativescript-loading-indicator": "^2.3.2",
    "nativescript-local-notifications": "^1.2.1",
    "nativescript-platform-css": "^1.4.0",
    "nativescript-plugin-firebase": "^4.0.2",
    "nativescript-pulltorefresh": "^2.0.1",
    "nativescript-push-notifications": "^0.1.2",
    "nativescript-sidedrawer": "^1.0.6",
    "nativescript-theme-core": "~1.0.2",
    "reflect-metadata": "~0.1.8",
    "rxjs": "~5.3.0",
    "tns-core-modules": "^3.1.0",
    "zone.js": "~0.8.2"
```
**TNS Paltform**
3.1.0

_Copied from original issue: NativeScript/NativeScript#4479_

### PR DESCRIPTION
**Pull request by sis0k0**
fixes #872

### CODE REVIEW

File: nativescript-angular/directives/list-view-comp.ts [hunk 1]
```diff
@@ -217,7 +217,7 @@ export class ListViewComponent implements DoCheck, OnDestroy, AfterContentInit {
 }
 
 function getSingleViewRecursive(nodes: Array<any>, nestLevel: number): View {
```
Comment (vakrilov) [inline nativescript-angular/directives/list-view-comp.ts]: Can we make this function public and export it so that it can be consumed from 3rd parties

**Comment by sis0k0** [comment]
run ci

**Comment by sis0k0** [comment]
sdkwebpack

**Comment by sis0k0** [comment]
sdkwebpack

**Comment by sis0k0** [comment]
sdkwebpack

**Pull request merged**